Resources/Prototypes/_SpaceCats/Entities/Objects/Specific/Medical/handheld_brigmedic_crew_monitor.yml

```
@@ -0,0 +1,31 @@
+  - type: WirelessNetworkConnection
+    range: 500
+
+- type: entity
+  id: HandheldBrigmedicCrewMonitorEmpty
+  parent: HandheldBrigmedicCrewMonitor
+  suffix: Empty
+  components:
+  - type: ItemSlots
+    slots:
+      cell_slot:
+        name: power-cell-slot-component-slot-name-default
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)